packages/tokens/package.json

```diff
@@ -30,7 +30,7 @@
   "devDependencies": {
     "@semantic-release/changelog": "^6.0.3",
     "@semantic-release/git": "^10.0.1",
-    "semantic-release": "^22.0.0",
+    "semantic-release": "^19.0.5",
```

Comment:
shouldn't this be a pinned version then?

Reply:
Well, it's just pinned to this major version and gets patch and minor fixes, so that's actually fine.